q2_types/feature_data/_type.py

```diff
@@ -52,13 +55,40 @@
                       variant_of=[FeatureData.field['type'],
                                   SampleData.field['type']])
 
+SequenceCharacteristics = SemanticType('SequenceCharacteristics',
+                                       variant_of=FeatureData.field['type'])
+
+
+@plugin.register_validator(FeatureData[SequenceCharacteristics %
+                                       Properties("length")])
+def validate_sequence_characteristics_length(data: pd.DataFrame, level):
```

Comment:
I don't want to be nit-picky but this function name is awfully long... Could we name it something like `validate_seq_char_len` maybe? 😅

Reply:
Sure :)